make_rolldev_file.py

```diff
@@ -0,0 +1,13 @@
+# For the pitch/roll table.  I used google sheets to save the xlsx file as
+# a CSV (after adding headers: pitch, negrolldev, rolldev).
```

Comment:
Better to read the excel file directly.
```
In [12]: df = pandas.read_excel('/Users/aldcroft/Downloads/pitch_roll_2022.xlsx'
    ...: , names=['pitch', 'rolldev_minus', 'rolldev_plus'])

In [13]: df
Out[13]: 
            pitch  rolldev_minus  rolldev_plus
0       46.612288      -0.060745      0.060745
1       46.622288      -0.116114      0.116114
2       46.632288      -0.171483      0.171483
3       46.642288      -0.224661      0.224661
4       46.652288      -0.274136      0.274136
...           ...            ...           ...
13334  179.952288     -18.748772     18.748772
13335  179.962288     -18.748772     18.748772
13336  179.972288     -18.748772     18.748772
13337  179.982288     -18.748772     18.748772
13338  179.992288     -18.748772     18.748772

[13339 rows x 3 columns]

In [14]: t = Table.from_pandas(df)

In [15]: t
Out[15]: 
<Table length=13339>
      pitch           rolldev_minus         rolldev_plus   
     float64             float64              float64      
------------------ -------------------- -------------------
 46.61228812838714 -0.06074530362457799 0.06074530362457799
 46.62228812838715 -0.11611406766359549 0.11611406766359549
46.632288128387145  -0.1714828317025736  0.1714828317025736
 46.64228812838714 -0.22466119433237589 0.22466119433237589
 46.65228812838714 -0.27413563619862313 0.27413563619862313
 46.66228812838715  -0.3236100780649055  0.3236100780649055
               ...                  ...                 ...
179.93228812838714  -18.748771807903847  18.748771807903847
179.94228812838713  -18.748771807903516  18.748771807903516
179.95228812838715  -18.748771807903186  18.748771807903186
179.96228812838714  -18.748771807902855  18.748771807902855
179.97228812838713  -18.748771807902525  18.748771807902525
179.98228812838715  -18.748771807902195  18.748771807902195
179.99228812838714  -18.748771807901864  18.748771807901864
```

Reply:
It is better, but I don't see the point of the rework at this point.

Reply:
Well at least document the better way for future updates, which are 100% certain to happen.

Reply:
Good idea.  Thanks!